.github/workflows/deploy_artifacts.yml

```diff
@@ -1,15 +1,25 @@
-name: Deploy develop snapshot
-
+name: Deploy artifacts
 on:
   push:
-    branches:
-      - develop
-      
-  # Allows you to run this workflow manually from the Actions tab
-  workflow_dispatch:
+    tags:
+      - '**'
+    # Run for all branches except the following
+    branches-ignore:
+      - 'master'
+      - 'release/**'
+      - 'hotfix/**'
+      - 'dependabot/**'
   
 jobs:
-  build:
+  deploy_artifacts:
+    uses: dockstore/workflow-actions/.github/workflows/deploy_artifacts.yaml@seab-6860/image-changelist
```

Comment:
I think we should start creating tags in `dockstore/workflow-actions`, and the consumers of the actions should always reference the actions by tag. It's how we use 3rd party actions, e.g., `uses: actions/checkout@v4`. 

I don't know that it's necessary for this PR, especially since it's a revert, but we should look at that going forward.